include/igl/cross_field_mismatch.cpp

```diff
@@ -78,7 +69,7 @@ namespace igl {
 
 
 public:
-  inline MissMatchCalculator(const Eigen::PlainObjectBase<DerivedV> &_V,
+  inline MismatchCalculator(const Eigen::PlainObjectBase<DerivedV> &_V,
```

Comment:
Super nitpick: Above you aligned the arguments in a column (see mismatchByCross), can you do the same here?

Reply:
Is it fine?